Makefile

```diff
@@ -165,9 +168,14 @@ build: generate fmt vet ## Build manager binary.
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
-docker-build:
+docker-build-operator:
 	docker buildx build --load -t ${IMG} .
 
+docker-build-secrets-webhook:
+	docker buildx build -t ${WEBHOOK_IMG} -f cmd/secrets-webhook/Dockerfile .
+
+docker-build: docker-build-operator docker-build-secrets-webhook
```

Comment:
Running this target doesn't seem to leave me with a `k8ssandra/k8ssandra-secrets-webhook:latest` image in my local docker registry. I think you might need to add `--load` to the `docker-build-secrets-webhook` target to make this work.